.github/workflows/compile_to_uf2.yml

```diff
@@ -0,0 +1,70 @@
+name: Create uf2 release for new version tag
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  compile-uf2:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: clone EuroPi
+        uses: actions/checkout@v3
+        with:
+          path: europi
+
+      - name: clone micropython
+        uses: actions/checkout@v3
+        with:
+          repository: micropython/micropython
+          ref: v1.19.1
+          path: micropython
+      
+      - name: install os deps
+        run: sudo apt-get install -y cmake gcc-arm-none-eabi libnewlib-arm-none-eabi build-essential
+
+      - name: prepare micropython build
+        working-directory: micropython
+        run: make -C ports/rp2 submodules && make -C mpy-cross
+
+      - name: copy europi into micropython modules
+        run: |
+          mkdir micropython/ports/rp2/modules/contrib
+          mkdir micropython/ports/rp2/modules/experimental
+          cp -r europi/software/firmware/*.py micropython/ports/rp2/modules
+          cp -r europi/software/firmware/experimental/*.py micropython/ports/rp2/modules/experimental
+          cp -r europi/software/contrib/*.py micropython/ports/rp2/modules/contrib
```

Comment:
[Discussion] What is the intended workflow for someone who installs the software this way? My assumption is that, in our supported use case, they would only ever use the `uf2` files. They would never copy individual scripts over. If that's not the case we'll need a bunch more testing to know how this thing behaves when copied scripts clash with included scripts.

Either way, I think we'll need some documentation explaining the expected use of this release, as well as upgrade paths. I don't think that the documentation necessarily needs to be in this PR, but it should be in whatever firmware release includes this PR. That way we have something to point people to when questions come up.

Reply:
Yeah, this definitely changes the getting started / programming instructions. Agreed that it'd be ideal but out of scope for this PR.